packages/remix-dev/package.json

```diff
@@ -64,6 +64,7 @@
     "@types/gunzip-maybe": "^1.4.0",
     "@types/inquirer": "^8.2.0",
     "@types/jscodeshift": "^0.11.3",
+    "@types/jsesc": "^3.0.1",
```

Comment:
this should've been here already, also bumped the version to match the actual major version of `jsesc` being used here

Reply:
Can't we just delete it here? 🤔

Reply:
it's being used [here](https://github.com/remix-run/remix/blob/c0cf73a1995a39ce064942aca3d5b3d89f6ec404/packages/remix-dev/compiler/plugins/serverAssetsManifestPlugin.ts#L38) in the `serverAssetsManifestPlugin` esbuild plugin. i'm not sure why or what it's needed for though so i've just opted to leave it as is.